.env.frontend.example

```
@@ -2,3 +2,5 @@ REACT_APP_API_URL={API_BASE_URL}
 REACT_APP_AUTH_ENV={AUTH_ENV}
 REACT_APP_FEEDBACK_URL={FEEDBACK_URL}
 REACT_APP_INTERCOM_APP_ID={APP_ID}
+REACT_APP_SENTRY_ENV={SENTRY_ENVIRONMENT}
```

So we will have a `staging` and a `production` environment such that we can keep track of errors separately. And just ignore errors that are neither of those two environments (like development) so we do away with the inbound/localhost filters. Is that what you had in mind?

Let's deploy this to staging tomorrow and set up the staging environment in Sentry to test out that env setup.